src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -400,6 +422,7 @@ protected ResponseEntity<Object> searchUsersByOrganisation(String organisationId
         showDeleted = getShowDeletedValue(showDeleted);
         returnRoles = getReturnRolesValue(returnRoles);
 
+        //copy from here
```

Comment:
Can you remove this comment?

Reply:
Sorry, forgot to remove it - thanks for spotting it @SujeethKumarAri

Reply:
no problem :)